lib/dfe/analytics.rb

```diff
@@ -119,6 +119,12 @@ def self.blocklist
       Rails.application.config_for(:analytics_blocklist)
     end
 
+    def self.custom_events
+      Rails.application.config_for(:analytics_custom_events)
+    rescue RuntimeError
```

Comment:
Bad Rails not returning specific exception type in this context!

Reply:
It's painful, but I think I'd rather live with generic exception than break the abstraction of `config_for`? wdyt?